lib/index.js

```diff
@@ -113,7 +227,10 @@ Attribution.prototype.onidentify = function(identify) {
  */
 
 Attribution.prototype.ontrack = function(track) {
-  this.send('/track', track.json());
+  var json = track.json();
+  // TODO: figure out why we need traits.
+  delete json.traits;
```

Comment:
@mgz we don't need it, it's left over from early implementation, safe to remove this delete